docs/VALUES.md

```diff
@@ -0,0 +1,32 @@
+# Core Values
+Shared by a broader community of the Organic Maps project developers and users.
+
+- [Free and Open Source](#free-and-open-source)
+- [Privacy](#privacy)
+- [Community](#community)
+- Core qualities of user-facing Organic Maps apps:
+  - Easy to use
+  - Offline-centric
+  - Fast and light on resources
+  - Privacy-respecting
+
+# Free and Open Source
+The complete Organic Maps project codebase and all associated tools, documents and resources are released as free-as-in-freedom software using Free Software licenses.
+All data comes from open sources (e.g. map data is based on OpenStreetMap).
+
+*At the moment non-F-Droid Android builds of OM contain a proprietary Google Play Services Location library, which possible could be replaced by a FOSS implementation in the future.
+Please let us know if you spot any other non-free bits.
+
+**The data file associating OSM hotel POIs with Kayak listings is courtesy of Kayak.com.
+
+# Privacy
+All personal information is protected by default to a reasonable extent.
+There are additional options for sensitive users to further enhance protection by sacrificing some convenience / ease of use.
+No personal information is collected without user's explicit consent.
+Organic Maps might collect anonymous usage statistics in a privacy-preserving way. These stats couldn't possibly be used to identify or track a user.
+
+The project pledges to disclose all potential limitations on data protection and cases of anonymous data collection.
+
+# Community
+The Organic Maps project is based on a community of volunteer contributors and strives to be user-centric.
+The project endeavours to have little hierarchy in its governance and to conduct as much discussion in public as possible.
```

Comment:
Personally I'd like it to be
"All communication channels and documents are public. Private data (names, bank account number, etc.) and credentials (passwords, etc.) are the only data stored in a private space and redacted from documents."
but at the moment we're far from being there..

Reply:
Yeap this is about development-related discussions, not compliance etc.

The current version "The project endeavours to have little hierarchy in its governance and **to conduct as much discussion in public as possible.**" is short and good enough?

Reply:
By doing this process here, this limits contribution to primarily developers, and excludes non-developers.

Reply:
@oleg-rswll do you have a suggestion how to make it better?

I thought first it would be better to make a draft based mostly on input from contributors. Once its shaped I planned to make wider announcements in user chats.

Reply:
A document on a platform where more people can contribute - where more people have accounts, or where an account may not be required, and easier to engage for a non-technical user base, would help include more people.  An example can be Google Docs as an option for this purpose (although I do not endorse Google in general).

Reply:
Ok let's try it :) https://board.net/p/Organic_Maps_Values